lib/prototyp/bigint.go

```diff
@@ -219,36 +246,13 @@ func (b *BigInt) Scan(src interface{}) error {
 	}
 
 	*b = BigInt(*i)
-
-	return nil
-}
-
-func (b *BigInt) ExtensionType() int8 {
```

Comment:
note: cbor/v2 doesn't use these methods anymore.. it just uses MarshalBinary and UnmarshalBinary